index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
```

Comment:
It helps a browser and search engine to understand your current localisation. 
```suggestion
<html lang="ru">
```

Reply:
+

Reply:
?

Reply:
What does the dot mean?)